javav2/example_code/rekognition/src/main/java/com/example/rekognition/RotateImage.java

```diff
@@ -0,0 +1,141 @@
+
+public class RotateImage {
+
+    public static void main(String[] args) {
+
+        final String USAGE = "\n" +
+                "Usage: " +
+                "RecognizeCelebrities <sourceImage>\n\n" +
+                "Where:\n" +
+                "sourceImage - the path to the image (for example, C:\\AWS\\pic1.png). \n\n";
+
+       if (args.length != 1) {
+            System.out.println(USAGE);
+            System.exit(1);
+       }
+
+        String sourceImage = args[0];
+        Region region = Region.US_EAST_2;
+        RekognitionClient rekClient = RekognitionClient.builder()
+                .region(region)
+                .build();
+
+        System.out.println("Locating celebrities in " + sourceImage);
+        recognizeAllCelebrities(rekClient, sourceImage);
+        rekClient.close();
+    }
+
+    // snippet-start:[rekognition.java2.recognize_image orientation.main]
+    public static void recognizeAllCelebrities(RekognitionClient rekClient, String sourceImage) {
+
+        try {
+            BufferedImage image = null;
+            InputStream sourceStream = new FileInputStream(new File(sourceImage));
+            SdkBytes sourceBytes = SdkBytes.fromInputStream(sourceStream);
+
+            image = ImageIO.read(sourceBytes.asInputStream());
+            int height = image.getHeight();
+            int width = image.getWidth();
+
+            Image souImage = Image.builder()
+                    .bytes(sourceBytes)
+                    .build();
+
+            RecognizeCelebritiesRequest request = RecognizeCelebritiesRequest.builder()
+                    .image(souImage)
+                    .build();
+
+            RecognizeCelebritiesResponse result = rekClient.recognizeCelebrities(request) ;
+
+            List<Celebrity> celebs=result.celebrityFaces();
+            System.out.println(celebs.size() + " celebrity(s) were recognized.\n");
+
+            for (Celebrity celebrity: celebs) {
+                System.out.println("Celebrity recognized: " + celebrity.name());
+                System.out.println("Celebrity ID: " + celebrity.id());
+                ComparedFace  face = celebrity.face();
+                ShowBoundingBoxPositions(height,
+                        width,
+                        face.boundingBox(),
+                        result.orientationCorrectionAsString());
+            }
+
+        } catch (RekognitionException | FileNotFoundException e) {
+            System.out.println(e.getMessage());
+            System.exit(1);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void ShowBoundingBoxPositions(int imageHeight, int imageWidth, BoundingBox box, String rotation) {
+
+        float left = 0;
+        float top = 0;
+
+        if(rotation==null){
+            System.out.println("No estimated estimated orientation. Check Exif data.");
+            return;
+        }
+        //Calculate face position based on image orientation.

```

Comment:
Leave one space between the start of the comment (//) and the comment (everywhere).

Reply:
fixed